stream-sdk/stream-parent/stream-worker/pom.xml

```diff
@@ -35,11 +35,6 @@
             <artifactId>spring-cloud-starter-sleuth</artifactId>
         </dependency>
 
-        <dependency>
-            <groupId>org.springframework.cloud</groupId>
-            <artifactId>spring-cloud-sleuth-zipkin</artifactId>
-        </dependency>
```

Comment:
maybe we should remove this explicit dependency only by the time we migrate to SSDK parents as the services heavily rely on it for traceability